apps/build-onchain-apps/src/pages/home/components/FormBuyCoffee.tsx

```diff
@@ -77,8 +77,8 @@ function FormBuyCoffee({ onComplete }: FormBuyCoffeeProps) {
     <Box style={{ whiteSpace: 'nowrap' }}>
       <Flex direction="column">
         <Flex justify="center" position="relative">
-          <form onSubmit={handleSubmit}>
-            <Card size="4" style={{ width: 400 }}>
+          <form onSubmit={handleSubmit} style={{ width: '100%' }}>
```

Comment:
Why these CSS changes? How the before and after look like?

Reply:
@Zizzamia updated the PR description